docs/guide/markdown.md

```diff
@@ -233,7 +233,7 @@ This is a special container that can be used to prevent style and router conflic
 
 ```md
 ::: raw
-Wraps in a <div class="vp-raw">
+Wraps in a <div class="vp-raw" />
```

Comment:
ah it should be `Wraps in a <div class="vp-raw">`

Reply:
I see, thank you for your response.